drivers/storage/portworx/components.go

```diff
@@ -93,9 +101,12 @@ const (
 )
 
 var (
-	kbVerRegex     = regexp.MustCompile(`^(v\d+\.\d+\.\d+).*`)
-	controllerKind = corev1alpha1.SchemeGroupVersion.WithKind("StorageCluster")
-	specsBaseDir   = getSpecsBaseDir
+	kbVerRegex            = regexp.MustCompile(`^(v\d+\.\d+\.\d+).*`)
+	controllerKind        = corev1alpha1.SchemeGroupVersion.WithKind("StorageCluster")
+	specsBaseDir          = getSpecsBaseDir
+	autopilotConfigParams = map[string]string{
```

Comment:
Keeping this helps people overwrite the value using args section in autopilot. If we remove this then it will simply be passed to autopilot as an arg (if someone sets it in the storage cluster spec). 

I will change the code to just use it for finding if any config param is present in the autopilot args. Will not pass in the default config.